ui/src/app/components/copy-modal.tsx

```diff
@@ -21,54 +24,127 @@ const ResourceTypeHomeTabs = new Map()
   .set(ResourceType.DATASET, 'data');
 
 export interface Props {
+  cdrVersionListResponse: CdrVersionListResponse;
   fromWorkspaceNamespace: string;
-  fromWorkspaceName: string;
+  fromWorkspaceFCName: string;
```

Comment:
to clarify that this is not the user-friendly `name` with capitalization and spaces